distribution/changelog.txt

```diff
@@ -4,6 +4,7 @@
 - Fix: [#1972, #11679] Vehicles passing by toilets can cause them to glitch.
 - Fix: [#9999, #10000, #10001, #10002, #10003] Truncated scenario strings when using Catalan, Czech, Japanese, Polish or Russian.
 - Fix: [#16357] Chairlift station covers draw incorrectly.
+- Fix: [#18433] Chinese characters displayed as ??? when environment var LANG not properly set.
```

Comment:
This is too general, as it suggests this always happened. What about the following?
```suggestion
- Fix: [#18433] CJK TrueType fonts could not be located when font names are translated by the OS.
```